internal/pkg/scaffold/ansible/dockerfilehybrid.go

```diff
@@ -62,16 +62,20 @@ ENV OPERATOR=/usr/local/bin/ansible-operator \
 # Ensure fresh metadata rather than cached metadata in the base by running
 # yum clean all && rm -rf /var/yum/cache/* first
 RUN yum clean all && rm -rf /var/cache/yum/* \
- && yum install -y https://dl.fedoraproject.org/pub/epel/epel-release-latest-7.noarch.rpm \
- && yum install -y python-devel gcc inotify-tools \
- && easy_install pip \
- && pip install -U --no-cache-dir setuptools pip \
+ && (yum install -y https://dl.fedoraproject.org/pub/epel/epel-release-latest-7.noarch.rpm || true) \
+ && yum -y update \
+ && yum install -y python36-devel python36-pip gcc \
+ # Install inotify-tools. Note: rpm -i will install the rpm in the registry for allow yum install it.
+ && curl -O https://rpmfind.net/linux/fedora/linux/releases/30/Everything/x86_64/os/Packages/i/inotify-tools-3.14-16.fc30.x86_64.rpm \
+ && rpm -i inotify-tools-3.14-16.fc30.x86_64.rpm \
+ && yum install inotify-tools \
+ && pip3 install --upgrade setuptools pip \
  && pip install --no-cache-dir --ignore-installed ipaddress \
-      ansible-runner==1.2 \
+      ansible-runner==1.3.4 \
       ansible-runner-http==1.0.0 \
       openshift==0.8.9 \
-      ansible==2.8 \
- && yum remove -y gcc python-devel \
+      ansible==2.8.5 \
```

Comment:
We shouldn't pin to a Z version here, z-stream is backwards compatible and will contain security and bugfixes

Reply:
If we leave with 2.8 the version installed is 2.8.0 and not 2.8.5 which is the latest one :-(

Reply:
Ah, ok then we should do `ansible~=2.8.0` which will install 2.8.*